packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/generateDataURL.ts

```diff
@@ -0,0 +1,49 @@
+import getGeneratedImageSize from './getGeneratedImageSize';
+import ImageEditInfo from '../types/ImageEditInfo';
+
+/**
+ * @internal
+ * Generate new dataURL from an image and edit info
+ * @param image The image to generate data URL from. It is supposed to have original src loaded
+ * @param editInfo Edit info of the image
+ * @returns A BASE64 encoded string with image prefix that represents the content of the generated image.
+ * If there are rotate/crop/resize info in the edit info, the generated image will also reflect the result.
+ * It is possible to throw exception since the original image may not be able to read its content from
+ * the code, so better check canRegenerateImage() of the image first.
+ */
+export default function generateDataURL(image: HTMLImageElement, editInfo: ImageEditInfo): string {
+    const {
+        angle,
+        width,
+        height,
+        bottom,
+        left,
```

Comment:
left/right here actually indicates the distance as percentage from the left/right border of the image, correct? could we give them a different name as it would make people think that's the position of the left/right border but it is not. maybe something like distFromLeft/distFromRight. Same as top/bottom.

Reply:
Actually here I'm trying to save some bytes since those names won't be compressed.